src/coreclr/jit/rationalize.cpp

```diff
@@ -398,23 +398,20 @@ void Rationalizer::RewriteAssignment(LIR::Use& use)
 #ifdef FEATURE_SIMD
         if (varTypeIsSIMD(location) && assignment->OperIsInitBlkOp())
         {
-            if (location->OperGet() == GT_LCL_VAR)
+            if (location->OperIs(GT_LCL_VAR))
             {
-                var_types   simdType        = location->TypeGet();
-                GenTree*    initVal         = assignment->AsOp()->gtOp2;
-                CorInfoType simdBaseJitType = comp->getBaseJitTypeOfSIMDLocal(location);
-                if (simdBaseJitType != CORINFO_TYPE_UNDEF)
-                {
-                    GenTreeSIMD* simdTree = new (comp, GT_SIMD)
-                        GenTreeSIMD(simdType, initVal, SIMDIntrinsicInit, simdBaseJitType, genTypeSize(simdType));
-                    assignment->AsOp()->gtOp2 = simdTree;
-                    value                     = simdTree;
-                    initVal->gtNext           = simdTree;
-                    simdTree->gtPrev          = initVal;
-
-                    simdTree->gtNext = location;
-                    location->gtPrev = simdTree;
-                }
+                var_types    simdType        = location->TypeGet();
+                GenTree*     initVal         = assignment->AsOp()->gtOp2;
+                CorInfoType  simdBaseJitType = comp->getBaseJitTypeOfSIMDLocal(location);
+                GenTreeSIMD* simdTree        = new (comp, GT_SIMD)
```

Comment:
We should prefer `gtNewSimdNode` since it may do additional handling (now or in the future).